### ISSUE TITLE
Position indication does not work properly in new uppercase letter detection

### ISSUE THREAD

**Issue by theseion** [coreruleset/crs-toolchain#119]
The indication of position (`======^ [HERE]`) for the new uppercase letter detection doesn't work properly.

1. For a match at position 0, the repeat count will be illegal (-1)
2. The position indication depends on the logger output, which in turn depends on name and location of the binary. For example:
```shell
06:39:54 WRN ../../../go/pkg/mod/github.com/coreruleset/crs-toolchain@v1.4.3-0.20240301053521-34f5270a4d17/cmd/regex_format.go:211 >   ##!> include-except unix-shell-4andup unix-shell-fps-pl1 -- @ [\s<>&|)] ~ \S
============================================================================^ [HERE]
```

**Comment by fzipi** [comment]
Yeah, thought about (1) but forgot to create a test case. My bad.

**Comment by theseion** [comment]
Fixed by #120.
